src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyAmendmentServiceTests.cs

```diff
@@ -34,7 +37,8 @@ public async Task CreateMethodologyAmendment_ClonesBasicFields()
 
             // creation and update fields
             Created = DateTime.Today.AddDays(-2),
-            CreatedBy = new User(),
+            CreatedBy = _dataFixture.DefaultUser()
+                .Generate(),
```

Comment:
Minor one, but you probably don't need any of the `.Generate()` lines added here throughout this class.

It should use the implicit conversion from Generator<User> to User.